desc/io/input_reader.py

```diff
@@ -776,6 +778,166 @@ def write_desc_input(filename, inputs, header=""):  # noqa: C901 - fxn too compl
 
         f.close()
 
+    @staticmethod
+    def descout_to_input(  # noqa: C901 - fxn too complex
+        filename,
+        inputs,
+        header="#DESC-generated input file",
+        ftol=1e-2,
+        xtol=1e-6,
+        gtol=1e-6,
+        maxiter=100,
+        optimizer="lsq-exact",
+        objective="force",
+    ):
+        """Generate a DESC input file from a DESC output file.
+
+        Parameters
+        ----------
+        filename : str or path-like
+            name of the file to create
+        inputs : str or path-like
+            path of the DESC output equilibrium file
+        header : str
+            text to print at the top of the file
+
+        DESC will automatically choose continuation parameters unless you
+        have specified those prameters as kwargs
```

Comment:
the signature doesn't take kwargs, and it looks like the code doesn't have stuff for dealing with non-default continuation params? That's fine as long as we fix the docstring to match. Also usually extended description like this goes before the `Parameters` block, or as a seperate `Notes` block (with `------` after)

Reply:
I tried kwargs, but that means we should check whether to use kwargs provided by the user or the default values. I can't think how to do that without putting if statements for each key in kwargs. More if statements => more tests. I don't want to decrease coverage. 

I'll edit the docstrings.

Reply:
I think the most important kwarg there is objective. We should definitely set the right objective in the input file.